scripts/README.md

```diff
@@ -54,21 +54,20 @@
 
 - <span><b>chairman.sh</b></span> <br/>  Run the Chairman consensus checker against a cluster started
                      by `shelley-testnet*.sh` family of scripts
-- <span><b>generator.sh</b></span> <br/> Run the transaction generator against a cluster started
-                     by `shelley-testnet` family of scripts
 - <span><b>genesis.sh</b></span> <br/> Generate a new genesis in the `configuration/` folder
-- <span><b>get-default-key-address.sh</b></span> <br/> Given a signing key, print its associated address
```

Comment:
This script was removed, but its functionality lives on as a shell function in `scripts/lib-cli.sh`